Cargo.toml

```
@@ -18,29 +18,39 @@ zeroize = { version = "1.5", features = ["zeroize_derive"] }
 hpke-rs-crypto = { version = "0.3.0-alpha.1", path = "./traits", default-features = false }
 rand_core = { version = "0.9", default-features = false }
 libcrux-sha3 = { version = "0.0.2" }
+hpke-rs-rust-crypto = { version = "0.3.0-alpha.1", path = "./rust_crypto_provider", default-features = false, optional = true }
+hpke-rs-libcrux = { version = "0.2.0-alpha.1", path = "./libcrux_provider", default-features = false, optional = true }
```

One small nitpick for this change I added yesterday is that these crates do not have default features set, so there is not a need to disable the default features here. 
```suggestion
hpke-rs-rust-crypto = { version = "0.3.0-alpha.1", path = "./rust_crypto_provider", optional = true }
hpke-rs-libcrux = { version = "0.2.0-alpha.1", path = "./libcrux_provider", optional = true }
```
In particular, the `"std"` feature already would need to be explicitly enabled on these crates for std support, which is already gated behind the `"std"` feature on this crate.

---

Yeah, but it also doesn't do anything. If anything, it's safer to disable because nothing that may get added gets enabled by default. Leaving as is to move this along.